tests/data_context/store/test_datasource_store.py

```diff
@@ -68,6 +71,7 @@ def test_datasource_store_with_bad_key_raises_error(
 
 def _assert_serialized_datasource_configs_are_equal(
     datasource_configs: List[DatasourceConfig],
+    serializers: Optional[List[AbstractConfigSerializer]] = None,
```

Comment:
Perhaps we pair config with serialized in a series of tuples? Good either way.

This could also be a nice place to use `zip()` to iterate through equally sized collections.